schematic/models/metadata.py

```diff
@@ -323,20 +326,20 @@ def submit_metadata_manifest(
             if not val_errors:
 
                 # upload manifest file from `manifest_path` path to entity with Syn ID `dataset_id`
-                syn_store.associateMetadataWithFiles(
+                syn_id = syn_store.associateMetadataWithFiles(
```

Comment:
Let's use manifest_id instead of syn_id. Similarly to other modules outside of store/synapse.py let's try to minimize references to Synapse.